app/views/admin/_admin_nav.html.erb

```diff
@@ -1,15 +1,14 @@
-<h3 class="landmark heading"><%= ts("Admin Navigation") %></h3>
+<h3 class="landmark heading"><%= t(".landmark") %></h3>
 <ul class="navigation actions" role="navigation">
   <li>
-    <%= span_if_current ts("AO3 News", key: "header"), admin_posts_path %>
+    <%= span_if_current t(".ao3_news"), admin_posts_path %>
    </li>
   <li>
-    <%= span_if_current ts("Post AO3 News", key: "header"),
-                        new_admin_post_path %>
-</li>
+    <%= span_if_current t(".post_ao3_news"), new_admin_post_path %>
```

Comment:
Not exactly related here, but since this file modification will introduce merge conflicts anyway and it's pretty minor: I think this needs to check `policy(AdminPost).can_post?`, similar to the header partial.